README.md

```diff
@@ -79,7 +79,7 @@ pip install lightning_tools
 __Usage:__
 
 ```bash
-python -m lightning_tools [group] [command]
+python -m lightning_tools.dev [group] [command]
```

Comment:
Can we describe how this `.dev` work? :)

Reply:
What do you mean exactly? It's the module name

Reply:
ok, get it :)